### PR TITLE
Attempting git push

### PR DESCRIPTION
**Pull request by BillyWu888**
👍

### CODE REVIEW

**Comment by lambdanerd** [comment]
Nice!

File: billy_wu/Python/Animal_Class.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        self.health = 150
+    def pet(self):
+        self.health = self.health + 5
+        return self
+dog=Dog("Boomer",100)
+dog.walk().walk().walk().run().run().pet().display_health()
+
+class Dragon(Animal):
+    def __init__(self,name,health):
+        super(Dragon,self).__init__(name,health)
```
Comment (ranriy) [inline billy_wu/Python/Animal_Class.py]: In python 3, super().__init__(name,health) also works

File: billy_wu/python_stack/flask_fundamentals/DojoSurvey/DojoSurvey.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+@app.route('/')
+def formpage():
+    return render_template('DojoSurvey.html')
+
+@app.route('/result', methods=['POST'])
+def user_info():
+    print("Got User Info")
+    name = request.form['name']
+    location = request.form['location']
+    language = request.form['language']
```
Comment (ranriy) [inline billy_wu/python_stack/flask_fundamentals/DojoSurvey/DojoSurvey.py]: Extra work!! If you are passing the dictionary user = request.form then there's no need to store the values in name, location, language etc

File: billy_wu/python_stack/flask_fundamentals/GreatNumberGame/GreatNumberGame.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    print(session)
+    return render_template('/GreatNumberGame.html')
+
+@app.route('/guess')
+def guess():
+    if 'guess' not in session:
+        session['guess'] = guess
+        print(session)
+    if session['guess'] >= randint:
+        return redirect('/', number=1)
```
Comment (ranriy) [inline billy_wu/python_stack/flask_fundamentals/GreatNumberGame/GreatNumberGame.py]: Hope you'll share the changed code too :)

File: billy_wu/python_stack/flask_fundamentals/hello_flask/alg.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+function invertHash(obj){
+    var newObj={};
+    for(i in obj){
+        newObj.obj[i]=i;
```
Comment (ranriy) [inline billy_wu/python_stack/flask_fundamentals/hello_flask/alg.js]: newObj[obj[i]]=i;  //this works. The dot gives an error

File: billy_wu/python_stack/myEnvironments/py3/lib/python3.7/distutils/__init__.py [hunk 5]
```diff
@@ -0,0 +1,101 @@
+import os
+import sys
+import warnings 
+import imp
+import opcode # opcode is not a virtualenv module, so we can use it to find the stdlib
+              # Important! To work on pypy, this must be a module that resides in the
```
Comment (ranriy) [inline billy_wu/python_stack/myEnvironments/py3/lib/python3.7/distutils/__init__.py]: Environments :(

**Comment by ranriy** [comment]
Don't upload the virtual env next time

**Pull request merged**